src/date-util/date-util.spec.ts

```diff
@@ -488,4 +488,44 @@ describe('DateUtil', () => {
       expect(DateUtil.secondsToTimeFormat(100_000)).toEqual('27:46:40'); // 100,000초 => 27시간 46분 40초
     });
   });
+
+  describe('formatInTwoDigitLocalTime', () => {
+    const testDate1 = '2000-01-01 00:00:00+09:00';
+    const testDate2 = '2000-01-01 12:00:00+09:00';
+    const testDate3 = '2000-01-01 12:00:00Z';
+    const testDate4 = new Date(testDate3).getTime();
+
+    it('should format date in two digit with year', () => {
+      const testOption1 = {
```

Comment:
```suggestion
      const testOption1: LocalDateTimeFormatOpts = {
```
지금은 LocalDateTimeFormatOpts의 모든 필드가 readonly라서 적용되지 않지만
이 타입선언을 넣냐 안넣냐의 차이가 큰게
withYear가 readonly가 아닌경우
delete testOption1.withYear;
를 했을 때 컴파일 에러가 나냐 안나냐의 차이가 있습니다.